packages/shared/components/atoms/tiles/TransactionActivityTile.svelte

```diff
@@ -110,6 +111,9 @@
     }
 
     function handleClaimClick(): void {
+        if (!$isSoftwareProfile) {
+            $isInternalTransaction = true
+        }
```

Comment:
Why is this only the case for Ledger profiles?

Reply:
Because we need a different `VerificationLedgerTransactionPopup` based on whether it's an internal transaction or not. I'll rename the prop to reflect the fact that it's only needed for ledger.